JustSaying.AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -44,19 +46,30 @@ public bool Subscribe(IAmazonSQS amazonSqsClient, SqsQueueBase queue)
             return false;
         }
 
-        public void Publish(Message message)
+        public async Task Publish(Message message)
```

Comment:
Maybe we should provide both, in which case it would be `Publish and `PublishAsync`. But that is  a larger PR, so not today.